### PR TITLE
Provide clarification on which mounted disk types are supported

### PR DESCRIPTION
**Pull request by evanphx**
Clarify that PTFE only supports block-style mounted disks.

### CODE REVIEW

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 1]
```diff
@@ -175,6 +175,24 @@ at install time and cannot be changed once the install is running.
 
 The decision you make will be entered during setup.
 
+#### Mounted Disk Guidelines
+
+The mounted disk option provides for significant flexibility in how PTFE stores it's data. In order to help customers narrow down possibilites we're providing some guidelines about how to use mounted disk.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/install-installer.html.md]: it's data > its data
In order to > To
~customers~
we're providing... > here are our guidelines about using mounted disk storage.

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 2]
```diff
@@ -175,6 +175,24 @@ at install time and cannot be changed once the install is running.
...
+
+The follow are **supported** mounted disk types:
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/install-installer.html.md]: ##### Supported mounted disk types
(as h5)

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 3]
```diff
@@ -175,6 +175,24 @@ at install time and cannot be changed once the install is running.
...
+* AWS EBS
+* GCP Zonal Persistent Disk
+* Azure Disk Storage
+* iSCSI
+
+The follow are **not supported** mounted disk types:
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/install-installer.html.md]: ##### Unsupported mounted disk types
(as h5)

File: content/source/docs/enterprise/private/install-installer.html.md (hunk 1 above)
Comment (nathanhruby) [inline content/source/docs/enterprise/private/install-installer.html.md]: its (no `)

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 4]
```diff
@@ -175,6 +175,24 @@ at install time and cannot be changed once the install is running.
...
+* NFS
+* SMB/CIFS
+
+The reason for supporting only some types of mounted disk is to provide relability and performance guarantees required to store and retrieve the data PTFE utilizes.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/install-installer.html.md]: The supported mounted disk types provide the necessary reliability and performance for data storage and retrieval in PTFE.

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 5]
```diff
@@ -175,6 +175,24 @@ at install time and cannot be changed once the install is running.
...
+
+If the type of mounted disk you wish to use is not here, please contact your HashiCorp representative to get clarification on if that type is supported.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/install-installer.html.md]: not here > not listed
on if > on whether

**Pull request merged**